spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -247,6 +255,7 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 			isDownStreamAppPartitioned = isPartitionedConsumer(appDeployTimeProperties,
 					upstreamAppSupportsPartition);
 
+			logger.info(String.format("Downloading resource URI [%s]", registration.getUri()));
```

Comment:
yeah, this is the important piece that we needed. I overlooked this earlier with my PR.